rosidl_generator_py/rosidl_generator_py/generate_py_impl.py

```diff
@@ -95,18 +97,27 @@ def generate_py(generator_arguments_file, typesupport_impls):
                     minimum_timestamp=latest_target_timestamp)
 
     for module in modules:
+        importlist = {}
+        for module_, type_ in modules[module]:
```

Comment:
The variable names `module_` and `module` don't provide enough information for readers to know what they contain / how they are different.